subcell_pipeline/simulation/readdy/parser.py

```diff
@@ -205,14 +209,23 @@ def parse_readdy_simulation_single_fiber_trajectory(
     return dataframe
 
 
+def round_2_sig_figs(x: float) -> int:
```

Comment:
Needs docstrings (or make private with `_` prefix and they won't show up in the docs)